app/routes/errors.py

```diff
@@ -52,7 +53,9 @@ def _render_error_page(status_code, template=None, **kwargs):
     handle_language()
     business_survey_config = get_survey_config(theme=SurveyType.BUSINESS)
     other_survey_config = get_survey_config(
-        theme=SurveyType.SOCIAL, base_url=ACCOUNT_SERVICE_BASE_URL_SOCIAL
+        theme=SurveyType.SOCIAL,
+        base_url=cookie_session.get("account_service_base_url")
+        or ACCOUNT_SERVICE_BASE_URL_SOCIAL,
     )
```

Comment:
Actually this won't work as the base url will be used by both types in this case. We actually don't need any changes in runner. You need to update the runner terraform and eq pipelines to add in `ACCOUNT_SERVICE_BASE_URL_SOCIAL`, at the moment, it doesn't set it, therefore it defaults to prod.